### PR TITLE
feature COR-1498-removed-toggle-and-vr-references

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard27**
## Summary

- Removed toggle button, keys, vr connections at  [vaccine-coverage-choropleth.tsx](https://github.com/minvws/nl-covid19-data-dashboard/compare/epic/COR-1475-safety-region-removal...feature/COR-1498-vaccinations-toggle-button-removal?expand=1#diff-682a0b5666a3b5a4d56ff6f94bf6174f38abd387c77eff6d36177fb6ab1372af)
- Merged together `vaccine-coverage-choropleth_vr_and_gm.tsx` to `vaccine-coverage-choropleth.tsx `
- Removed `vaccine-coverage-choropleth_vr_and_gm.tsx`

## Screenshots
**_Before_**
<details>

  <summary>Toggle before screenshot</summary> 

Screenshot 2023-03-29 at 13 55 27

</details>

**_After_**
<details>
  <summary>Toggle after screenshot</summary> 

Screenshot 2023-03-29 at 13 55 04

</details>

### CODE REVIEW

File: packages/app/src/domain/vaccine/vaccine-coverage-choropleth.tsx [hunk 1]
```diff
@@ -82,52 +78,28 @@ export const VaccineCoverageChoropleth = ({ data }: VaccineCoverageChoroplethPro
...
+        thresholds: thresholds.gm.primary_series_percentage,
         title: commonTexts.choropleth.choropleth_vaccination_coverage.shared.legend_title,
       }}
       metadata={{
         source: commonTexts.choropleth.vaccination_coverage.shared.bronnen.rivm,
-        date:
-          selectedMap === 'gm'
-            ? data.gm.find((item: GmCollectionVaccineCoveragePerAgeGroup | VrCollectionVaccineCoveragePerAgeGroup) => item.vaccination_type === selectedCoverageKind)?.date_unix
-            : data.vr.find((item: GmCollectionVaccineCoveragePerAgeGroup | VrCollectionVaccineCoveragePerAgeGroup) => item.vaccination_type === selectedCoverageKind)?.date_unix,
+        date: data.gm.find((item: GmCollectionVaccineCoveragePerAgeGroup | VrCollectionVaccineCoveragePerAgeGroup) => item.vaccination_type === selectedCoverageKind)?.date_unix,
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/domain/vaccine/vaccine-coverage-choropleth.tsx:86]: Maybe you can use the same solution as in the PR of COR-1489?

File: packages/app/src/domain/vaccine/vaccine-coverage-choropleth.tsx [hunk 2]
```diff
@@ -82,52 +78,28 @@ export const VaccineCoverageChoropleth = ({ data }: VaccineCoverageChoroplethPro
...
-            getLink: (gmcode) => reverseRouter.gm.vaccinaties(gmcode),
-          }}
-          formatTooltip={(context) => <ChoroplethTooltip data={context} ageGroups={matchingAgeGroups[selectedCoverageKind]} selectedCoverageKind={selectedCoverageKind} />}
-        />
-      )}
-
-      {selectedMap === 'vr' && (
-        <DynamicChoropleth
-          map={'vr'}
-          accessibility={{ key: 'vaccine_coverage_nl_choropleth' }}
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/domain/vaccine/vaccine-coverage-choropleth.tsx:118]: Is the key `vaccine_coverage_nl_choropleth` used in other places? If not, you might delete this key.

File: packages/cms/src/lokalize/key-mutations.csv [hunk 3]
```diff
@@ -49,3 +49,4 @@ timestamp,action,key,document_id,move_to
 2023-03-24T07:45:03.890Z,delete,pages.topical_page.shared.secties.search.title.nl,PGwiP4IwXrA0GfLCLD9Tth,__
 2023-03-24T07:45:03.890Z,delete,pages.topical_page.shared.secties.search.title.vr,7nClfXqPlyZyEU7PO2TAAw,__
 2023-03-23T08:50:33.174Z,delete,common.nav.links.veiligheidsregio,jF33EuwumlGuwav2FD3VW8,__
+2023-03-29T10:43:54.266Z,delete,common.choropleth.choropleth_vaccination_coverage.shared.vr,G6M7GhzH8XbtdKiRrez14Y,__
```
Comment (VWSCoronaDashboard21) [inline packages/cms/src/lokalize/key-mutations.csv:52]: Why is this key deleted?

Comment (VWSCoronaDashboard27) [reply packages/cms/src/lokalize/key-mutations.csv:52]: It was used in `commonTexts.choropleth.choropleth_vaccination_coverage.shared[selectedMap]` line 50 to select a region lines 51 and 115, I have changed it to `commonTexts.choropleth.choropleth_vaccination_coverage.shared.gm` so `shared.vr` is not needed.

Comment (VWSCoronaDashboard21) [reply packages/cms/src/lokalize/key-mutations.csv:52]: Great, thanks for clarifying this to me. I'll resolve this.

File: packages/app/src/domain/vaccine/vaccine-coverage-choropleth.tsx (hunk 1 above)
Comment (VWSCoronaDashboard21) [reply packages/app/src/domain/vaccine/vaccine-coverage-choropleth.tsx]: Nice! I'll resolve this.

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Great job! It looks good!

**Review by VWSCoronaDashboard18 — DISMISSED** [review]
Nice! Good job!

**Review by VWSCoronaDashboard26 — DISMISSED** [review]
A very nice solution and a great way to clean up a bit! 🙌🏻

**Pull request closed without merging**